### PR TITLE
Pin mixpanel to 2.2.0

### PR DESCRIPTION
**Pull request by girarda**
## What
Pin mixpanel because of issues with 2.3.0 https://airbytehq-team.slack.com/archives/C06H67UHTFX/p1715279312142039

## Can this PR be safely reverted and rolled back?
- [X] YES 💚

### CODE REVIEW

File: airbyte-integrations/connectors/source-mixpanel/metadata.yaml [hunk 1]
```diff
@@ -26,8 +26,10 @@ data:
   registries:
     cloud:
       enabled: true
+      dockerImageTag: 2.2.0
```
Comment (brianjlai) [inline airbyte-integrations/connectors/source-mixpanel/metadata.yaml:29]: can you add comment with a link to this PR for the pin

**Review by brianjlai — APPROVED** [review]
small thing otherwise lgtm

**Review by midavadim — APPROVED** [review]

**Comment by octavia-approvington** [comment]
Check that box <br/> TODO complete

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**